### PR TITLE
[CP-1713] Publish OS API as MVP to our community

### PR DESCRIPTION
**Pull request by lukasmas**
Publishing Protocol Description section to our community.
Publishing two endpoints: Device Info Endpoint and Contacts Endpoint.

**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: doc/os_api/endpoints/pure/device_info_endpoint.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        Discharging,
+        Charging,
+        PluggedNotCharging,
+    }
+};
+```
+```
+enum class SIM
+{
+    SIM1 =0,
```
Comment (dkarski) [inline doc/os_api/endpoints/pure/device_info_endpoint.md:158]:
```suggestion
    SIM1 = 0,
```

**Review by dkarski — APPROVED** [review]
❤️ ✅

File: doc/os_api/endpoints/pure/device_info_endpoint.md (hunk 1 above)
Comment (lukasmas) [reply doc/os_api/endpoints/pure/device_info_endpoint.md]: Done

**Review by marziel — APPROVED** [review]

File: doc/os_api/endpoints/pure/contacts_endpoint.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+         {
+            "address":"Czeczota 6, 02-600 Warsaw, Poland",
+            "altName":"Abraham",
+            "email":"Abraham@example.com",
+            "blocked":false,
+            "favourite":true,
+            "ice":true,
+            "id":1,
+            "numbers":[
+               "500675127"
```
Comment (pawel-mudita) [inline doc/os_api/endpoints/pure/contacts_endpoint.md:74]: Please replace all phone numbers in these examples with ones that definitely do not belong to anyone (for eg. 123456789).

**Pull request merged**